src/constants/appConstants.tsx

```diff
@@ -106,30 +107,46 @@ export const CHANNEL_LIST: { label: string; value: string }[] = [
 export const DURATION_LIST: { label: string; value: number }[] = [
   { label: "10 Mins", value: 10 },
   { label: "15 Mins", value: 15 },
-  { label: "20 Mins", value: 20 },
   { label: "30 Mins", value: 30 },
   { label: "45 Mins", value: 45 },
   { label: "60 Mins", value: 60 },
 ];
 // Chart Constants
 export const CHART = {
+  FONT_FAMILY: "Arial, sans-serif, Helvetica Neue",
```

Comment:
Are we using it? 
and can we use the font family specified in the tailwind.config.js instead?